src/components/Authentication/OAuth.vue

```diff
@@ -17,7 +17,7 @@ const iconMap = {
   linkedin: { dark: true, icon: 'linkedin', color: '#283e4a' },
   bitbucket: { dark: false, icon: 'bitbucket' },
   box: { dark: true, icon: 'box_com', color: '#0071f7' },
-  globus: { dark: true, icon: 'globus', color: '#335a95' },
+  globus: { dark: true, icon: 'globe', color: '#335a95' },
```

Comment:
We should isolate this in its own PR since it's unrelated to the topic.

Reply:
I'm using the `globe` icon in the file browser, so I changed its name to something generic.   Should I create a duplicate icon and then remove the duplicate in another PR?

Reply:
Ah, I see. I'd actually like to keep the `globus` icon identifier separate from `globe`, even though they would point to the same thing right now. One is a brand and one is generic, so it makes sense to have two different semantic identifiers for them.